service/pipelines/config.go

```diff
@@ -25,14 +25,20 @@ var (
 		featuregate.WithRegisterFromVersion("v0.112.0"),
 		featuregate.WithRegisterDescription("Controls whether profiles support can be enabled"),
 	)
+	AllowNoPipelines = featuregate.GlobalRegistry().MustRegister(
+		"service.AllowNoPipelines",
+		featuregate.StageAlpha,
+		featuregate.WithRegisterFromVersion("v0.122.0"),
+		featuregate.WithRegisterDescription("Allow starting the Collector without starting any pipelines."),
+	)
```

Comment:
Is the plan to eventually stabilize this, or keep it as an option forever?

Reply:
I'm not sure yet. Ideally we eventually stabilize it, but I'm not sure how we want to enable this going forward. I figure for now it's okay to just have the feature gate and we can adjust course later.

Reply:
I think there is a reasonable chance that we would stabilize this. The main downside of stabilizing this is people getting confused about how to configure the Collector, so I guess we could think about some further validation (say, allowing this kind of configuration only when you have configured extensions that use OpAMP?), but that's something that we can figure out over time

Reply:
@bogdandrutu Is this okay with you? I'd like to get this in this week (before Monday's release) so I can have the OpAMP Supervisor start using it.

Reply:
It is ok...